README.md

```diff
@@ -491,6 +492,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Dheeraj Khushalani](https://github.com/DheerajKhush)
   - [Dhrumi Shah](https://github.com/dhrumishah)
   - [Dhruv Gajjar](https://github.com/Dhruv-Gajjar)
+  - [DHRUV-ADAVADKAR](https://github.com/adavadkardhruv13)
```

Comment:
Consider using lowercase

Reply:
Have made the required changes, Do check them.